Wikipedia/Code/SectionEditorWebView+ExecCommand.swift

```diff
@@ -0,0 +1,100 @@
+@objc enum CodeMirrorExecCommandType: Int {
+    case cursorUp
+    case cursorDown
+    case cursorLeft
+    case cursorRight
+    case undo
+    case redo
+}
+
+extension SectionEditorWebView {
+    // Won't need this when we don't need @objc for `execCodeMirrorCommand` - i.e. 'CodeMirrorExecCommandType' could just be string enum.
+    private func string(for type: CodeMirrorExecCommandType) -> String {
+        switch type {
+        case .cursorUp:
+            return "cursorUp"
+        case .cursorDown:
+            return "cursorDown"
+        case .cursorLeft:
+            return "cursorLeft"
+        case .cursorRight:
+            return "cursorRight"
+        case .undo:
+            return "undo"
+        case .redo:
+            return "redo"
+        }
+    }
+    
+    // Method for relaying various commands to codemirror - i.e. 'execCodeMirrorCommand(type: .cursorUp)'
+    @objc func execCodeMirrorCommand(type: CodeMirrorExecCommandType, completionHandler: (SectionEditorWebViewCompletionWithResultBlock)? = nil) {
+        evaluateJavaScript("window.wmf.execCommand(window.wmf.ExecCommandType.\(string(for: type)));", completionHandler: completionHandler)
+    }
+}
+
+
+
+/*
+TODO:
+- use separate methods instead of the exec command approach above
```

Comment:
@nambatee head's-up here are initial stub methods which i'm gonna play with wiring up to the test web view's buttons